public/static/js/distros.js

```diff
@@ -1,8 +1,9 @@
-mciModule.controller('DistrosCtrl', function($scope, $window, $location, $anchorScroll, mciDistroRestService) {
+mciModule.controller('DistrosCtrl', function($scope, $window, $location, $anchorScroll, $filter, mciDistroRestService) {
 
   $scope.readOnly = !$window.isSuperUser;
 
   $scope.distros = $window.distros;
+  $scope.containerPools = $window.containerPools;
```

Comment:
You don't need to set this if you don't need this in the HTML

Reply:
But don't I need to set it since I am using it in this JS file?

Reply:
You can access $window.containerPools in the js